net/kadnode/files/kadnode.config

```diff
@@ -0,0 +1,63 @@
+##
+## KadNode is a P2P DNS resolver.
+##
+## You need to configure dnsmasq to let it 
+## resolve the .p2p domain using KadNode:
+## 
+## uci add_list dhcp.@dnsmasq[0].server='/p2p/::1#3535'
+## uci commit dhcp
+##
+
+config kadnode
+	option enabled 0
+
+
+## ECC Key usage:
+## 1. Create public/secret key pair with `kadnode --bob-create-key /etc/kadnode_secret.pem`
+## 2. Put the secret key file on the router that you want to resolve to and use it for option bob_load_key.
+## 3. Use the public key hex output with .p2p attached on other devices to resovle to the router IP address via kadnode.
+
+## Secret key for public key links
+#	list bob_load_key '/ect/mynode.pem'
+
+
+## TLS usage:
+## For resolving domains, put credentials on the router and use option tls_client_cert.
+## For announcing domains, put the certitifacate and secret key on router and use option tls_server_cert.
+
+## Folder of CA certificates
+#	list tls_client_cert '/usr/share/ca-certificates/'
```

Comment:
that's not where the normal ca-certs get installed, does this really need custom paths?

Reply:
and if I were nit picking - spelling - certitifacate ?

Reply:
@ldir-EDB0 nice catch. Thanks.

Reply:
@karlp what would the correct patch be? Btw., this option is optional and only needed when certain features are used.

Reply:
if you're packaging for openwrt/lede, I'd expect you have been trying out this option?  I'd expect as a user to be able to uncomment this and have it "do theh right thing"  Try installing the ca-certificates package and see where they're installed.

Reply:
@karlp the certificates might be from package ca-certificates or self signed. But there should be no forced dependency on package ca-certificates.
I changed the comment and default path to make it easier for the user.